packages/wrangler/src/dev.tsx

```diff
@@ -895,7 +895,17 @@ async function getBindings(
 		unsafe: configParam.unsafe?.bindings,
 		logfwdr: configParam.logfwdr,
 		d1_databases: identifyD1BindingsAsBeta([
-			...configParam.d1_databases,
+			...(configParam.d1_databases || []).map((d1Db) => {
+				if (!d1Db.preview_database_id) {
+					throw new Error(
```

Comment:
IIRC, KV warns but doesn't error. Do you want the same behavior here?

Reply:
nah it throws
<img width="724" alt="image" src="https://user-images.githubusercontent.com/3822106/205042437-278a2ac0-4e23-45e2-9610-ecd4585d1657.png">